source/org/zfin/db/postGmakePostloaddb/1158/ZFIN-9240.sql

```diff
@@ -0,0 +1,22 @@
+--liquibase formatted sql
+--changeset cmpich:ZFIN-9240.sql
+
+insert into data_note
+select get_id_and_insert_active_data('DNOTE'),
+       dblink_linked_recid,
+       'ZDB-PERS-030520-3',
+       now(),
+       Concat(ensembl_remove_temp.ert_accession, ' has been obsoleted and removed by Ensembl. ', 'Foreign DB: ', fdb_db_display_name, ' DB_LINK ID ', db_link.dblink_zdb_id, ' DB_LINK note: ',
+              db_link.dblink_info)
+from ensembl_remove_temp,
+     db_link,
+     foreign_db_contains,
+     foreign_db
+where dblink_acc_num = ert_accession
+  and dblink_fdbcont_zdb_id = fdbcont_zdb_id
+  and fdb_db_pk_id = fdbcont_fdb_db_id;
+
+delete
+from db_link
+where dblink_acc_num in (select * from ensembl_remove_temp);
```

Comment:
Should we drop the ensembl_remove_temp table at the end?

Reply:
I need to inspect the table for now. I can remove the table in the next sprint.